source/common/runtime/runtime_features.cc

```diff
@@ -81,6 +81,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.preserve_upstream_date",
     "envoy.reloadable_features.stop_faking_paths",
     "envoy.reloadable_features.strict_1xx_and_204_response_headers",
+    "envoy.reloadable_features.tls_use_io_handle_bio",
```

Comment:
This should be documented somewhere. Can you add a release note for this and also update the PR description to mention it? This will help us track it for removal later.

Reply:
Done! Let me know if I missed something because this is my first time doing release notes updates.